src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/InputJavadocComment.javadoc

```diff
@@ -1,4 +1,4 @@
 <p>
 My {@code Class}
 </p>
-@see never
\ No newline at end of file
+@see never
```

Comment:
Yes, I think it should be impossible because a java file cannot have EOF following documentation.